client/src/components/Table/index.js

```diff
@@ -0,0 +1,23 @@
+import ReactTable from "react-table";
+import 'react-table/react-table.css'
+
+import React, { Component } from "react";
+
+class Table extends Component {
+  constructor(props) {
+    super(props);
+    }
+
+  render() {
+
+    return(
+
+     <ReactTable
+    data={this.props.data}
```

Comment:
This works but it would be good to get into the habit of destructuring your props to avoid repetition.
e.g. `const { data, columns } = this.props`

Reply:
done